File: backend/src/main/java/nl/han/ngi/projectportalbackend/core/models/Task.java
package nl.han.ngi.projectportalbackend.core.models;

import java.util.List;

public class Task {
    String uuid;
    String projectUuid;
    String title;
    int isDone;
    List<String> skills;
    public String getTitle() {
        return title;
    }

    public String getProjectUuid() {
        return projectUuid;
    }

    public void setProjectUuid(String projectUuid) {
        this.projectUuid = projectUuid;
    }

    public void setTitle(String title) {
        this.title = title;
    }

    public int getIsDone() {
        return isDone;
    }

    public void setIsDone(int done) {
        isDone = done;
    }

    public List<String> getSkills() {
        return skills;
    }

    public void setSkills(List<String> skills) {
        this.skills = skills;
    }

    public String getUuid() {
        return uuid;
    }

    public void setUuid(String uuid) {
        this.uuid = uuid;
    }
}

File: frontend/portal/src/app/types/task.ts
export interface Task {
    uuid: string;
    projectUuid: string;
    title: string;
    description: string;
    completed: boolean;
    category: string;
    skills: string[];
    assignedTo: string | null;
    dueDate: string;
    comments: string[];
  }
  
  export interface NewTask {
    title: string;
    description: string;
    category: string;
    skills: string[];
    dueDate: string;
    assignedTo?: string | null;
    comments?: string[];
  }
  